### ISSUE TITLE
Cap `Neural` learning rate

### ISSUE THREAD

**Issue by George3d6** [mindsdb/lightwood#750]
When searching for a lr we should probably avoid anything past 0.1 or so, benchmark and implement if accuracy doesn't decrease (+ time doesn't increase noticeably) or if accuracy increase /w the change. It's likely that lrs higher than 0.1 (realistically even higher than 0.01 - 0.001) will result in sub-optimal fine-tuning (but adding a special fine-tuning phase with low lr is the kind of complexity we should avoid adding for now)

**Comment by paxcema** [comment]
Done in #1150.
